requirements.txt

```diff
@@ -0,0 +1,2 @@
+autopep8
```

Comment:
Not sure autopep8 is a requirement for the project.  Seems more like something that is part of the dev env.